src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -175,27 +287,97 @@ public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
 
     private HashMap<String, Field> getFieldMappings() {
         HashMap<String, Field> fieldMappings = new HashMap<>();
-        fieldMappings.put("title", StandardField.TITLE);
-        fieldMappings.put("version", StandardField.VERSION);
+        fieldMappings.put("abstract", StandardField.ABSTRACT);
+        fieldMappings.put("date-released", StandardField.DATE);
         fieldMappings.put("doi", StandardField.DOI);
+        fieldMappings.put("keywords", StandardField.KEYWORDS);
         fieldMappings.put("license", BiblatexSoftwareField.LICENSE);
+        fieldMappings.put("message", StandardField.COMMENT);
         fieldMappings.put("repository", BiblatexSoftwareField.REPOSITORY);
+        fieldMappings.put("title", StandardField.TITLE);
         fieldMappings.put("url", StandardField.URL);
-        fieldMappings.put("abstract", StandardField.ABSTRACT);
-        fieldMappings.put("message", StandardField.COMMENT);
-        fieldMappings.put("date-released", StandardField.DATE);
-        fieldMappings.put("keywords", StandardField.KEYWORDS);
+        fieldMappings.put("version", StandardField.VERSION);
+
+        // specific to references and preferred-citation
+        fieldMappings.put("edition", StandardField.EDITION);
+        fieldMappings.put("isbn", StandardField.ISBN);
+        fieldMappings.put("issn", StandardField.ISSN);
+        fieldMappings.put("issue", StandardField.ISSUE);
+        fieldMappings.put("journal", StandardField.JOURNAL);
+        fieldMappings.put("month", StandardField.MONTH);
+        fieldMappings.put("notes", StandardField.NOTE);
+        fieldMappings.put("number", StandardField.NUMBER);
+        fieldMappings.put("pages", StandardField.PAGES);
+        fieldMappings.put("status", StandardField.PUBSTATE);
+        fieldMappings.put("volume", StandardField.VOLUME);
+        fieldMappings.put("year", StandardField.YEAR);
         return fieldMappings;
     }
 
     private List<String> getUnmappedFields() {
         List<String> fields = new ArrayList<>();
```

Comment:
Make map `public static final` to avoid initialization at each import.

(This also removes the method)